ui/src/app/settings/components/repos-list/repos-list.tsx

```diff
@@ -148,6 +150,10 @@ export class ReposList extends React.Component<RouteComponentProps<any>> {
                                 <div className='argo-form-row'>
                                     <FormField formApi={formApi} label='Enable LFS support' field='enableLfs' component={CheckboxField}/>
                                 </div>
+                                <div className='argo-form-row'>
+                                    <FormField formApi={formApi} label='Refspecs to be used for fetching revisions from the remote repository'
+                                               field='fetchRefspecs' component={TagsInputField}/>
```

Comment:
UI-wise, `TagsInputField` is rather a dumb solution in this case. This was the only ready-to-use component for handling string arrays that I could spot. At least it works...

Reply:
What about using `component={Text}`

Reply:
Well, then I'd need some custom logic to convert back and forth between a single string and a string array. Maybe we can just live with the tags input field right now? This is really a niche feature.